### PR TITLE
update for ui object scale9 state refresh

### PR DESCRIPTION
**Pull request by liamcindy**
@andyque Please review this pr

### CODE REVIEW

File: tests/cpp-tests/proj.win32/cpp-tests.vcxproj [hunk 1]
```diff
@@ -582,6 +582,11 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\cocos\2d\libcocos2d.vcxproj">
       <Project>{98a51ba8-fc3a-415b-ac8f-8c7bd464e93e}</Project>
+      <Private>true</Private>
```
Comment (zilongshanren) [inline tests/cpp-tests/proj.win32/cpp-tests.vcxproj]: Why change the project settings?

**Comment by liamcindy** [comment]
@andyque Please check this commit

**Comment by zilongshanren** [comment]
@minggo 
This PR could be merge after the CI is ready.

**Comment by zilongshanren** [comment]
This PR fixed https://github.com/cocos2d/cocos2d-x/issues/10111

**Pull request merged**